ASP.NET Core/Web/AspNetCoreTemplate.Web/Areas/Identity/Pages/Account/ConfirmEmail.cshtml.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace AspNetCoreTemplate.Web.Areas.Identity.Pages.Account
+{
+    using System;
+    using System.Threading.Tasks;
+
+    using AspNetCoreTemplate.Data.Models;
+    using Microsoft.AspNetCore.Authorization;
```

Comment:
Blank line before this namespace group

Reply:
Is this OK?

Reply:
https://github.com/NikolayIT/ASP.NET-MVC-Template/pull/36/commits/81219365355611ede7ee4cd2e11859572b1df8d6